lib/portal.js

```diff
@@ -9,6 +9,27 @@ const KEYCODES = {
 
 export default class Portal extends React.Component {
 
+  static defaultProps = {
+    onOpen: () => {},
+    onClose: () => {},
+    onUpdate: () => {},
+  };
+
+  static propTypes = {
+    className: React.PropTypes.string,
+    style: React.PropTypes.object,
+    children: React.PropTypes.element.isRequired,
+    target: React.PropTypes.instanceOf(window.HTMLElement),
```

Comment:
@luchkonikita I gave up create the element. If it doesn't exist I use the default prop (body).
Here is what I've modified from your code:
https://github.com/tajo/react-portal/compare/master...jusbrasil:master#diff-247298a69e22df733d1aac4bffc9fb06R169.

You can apply the [patch](https://github.com/tajo/react-portal/compare/master...jusbrasil:master.patch) on your branch to see the changes.